api/base/utils.py

```diff
@@ -49,11 +49,18 @@ def is_bulk_request(request):
     content_type = request.content_type
     return 'ext=bulk' in content_type
 
+
 def is_truthy(value):
-    return value in TRUTHY
+    if isinstance(value, bool) or value is None:
+        return value
+    return value.lower() in TRUTHY
```

Comment:
Add check for `1` and `0`

See: https://github.com/encode/django-rest-framework/blob/328591693d7a3e734ca5a740dddf85e11ccd208f/rest_framework/fields.py#L667-L688

Reply:
In addition add a comment here to elaborate the behavior before and after. e.g. before upgrade `yEs` is not truthy but now `yEs` is truthy.